### DISCUSSION TITLE
interaction between jwt Authorization List (Optional) with Authorization ACL lists

### DISCUSSION THREAD

**Discussion by jonathan-dev** [emqx/emqx#11789] [General]
here: https://www.emqx.io/docs/en/v5.2/access-control/authn/jwt.html#authorization-list-optional it says:

> TIP
Authorization (ACL) rules returned by JWT will be checked before all Authorizers. For details, see [Authorization](https://www.emqx.io/docs/en/v5.2/access-control/authz/authz.html).

Does this mean that the JWT ACL get overwritten by the Authorizers or the other way around?
Thanks in advanced

**Comment by thalesmg** [comment]
Hi @jonathan-dev .

It means that the rules defined in the JWT, if any, take precedence over other authorizers.  If a topic or topic filter matches one defined in the JWT, that rule is used.  If not, authorizers down the chain will be tried in order.

**Reply by jonathan-dev** [reply]
Thanks for the explanation :) I wasn't able to get that from the docs maybe it could be stated more clearly somewhere.
